week-5-practise/js/main.js

```diff
@@ -0,0 +1,66 @@
+////////Part 1//////////
+var jumbotron = document.querySelector(".jumbotron");
+var jumbotronButton = document.querySelector(".buttons").children;
+
```

Comment:
Maybe name this `jumbotronButtons` not `jumbotronButton` - it's an array.

I'd define `volunteerBtn` at the start, and `donateBtn`.  Much more readable than `jumbotronButton[1]`, etc, and if we have to reorder the buttons or add a new one at the front, we only have to change 2 lines of code, instead of 8.